wgengine/wgengine.go

```diff
@@ -36,7 +36,7 @@ type PeerStatus struct {
 // TODO(bradfitz): remove this, subset of ipnstate? Need to migrate users.
 type Status struct {
 	Peers      []PeerStatus
-	LocalAddrs []string // TODO(crawshaw): []wgcfg.Endpoint?
+	LocalAddrs []string // TODO(josharian): comma-separated string, to match wgcfg.Peer.Endpoints?
```

Comment:
the TODO was and is about what it should be, but what is it today pre-TODO?

Reply:
Took a stab at documenting this, and removed the TODO. If it still makes sense to do a bit later, I will, but it seems like maybe the better TODO to tackle is the "remove this struct" one.